tsconfig.json

```diff
@@ -1,5 +1,6 @@
 {
   "compilerOptions": {
+    "module": "esnext",
```

Comment:
this is needed so vscode doesn't complain about us doing dynamic imports without setting `module` to something like "esnext" or "commonjs", since vscode only knows about this tsconfig and not the cjs and esm ones which already have `module` set